lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -37,7 +37,8 @@ def setup_rabbitmq_connection_config
         config :vhost, :validate => :string, :default => "/"
 
         # Enable or disable SSL
-        config :ssl, :validate => :boolean, :default => false
+        # Specify TLS version if using TLS e.g "TLSv1.2"
+        config :ssl, :validate => :string, :default => false
```

Comment:
I don't think we should have a ':default' here. If none is specified it will be nill which is falsey. Having a 'string' that can contain a boolean is weird :)